internal/controller/admission/falconadmission_controller_test.go

```diff
@@ -168,6 +226,78 @@ var _ = Describe("FalconAdmission controller", func() {
 				}
 				return nil
 			}, time.Minute, time.Second).Should(Succeed())
+
+			By("Deleting FalconAdmission")
+			Eventually(func() error {
+				return k8sClient.Delete(ctx, falconAdmission)
+			})
+			Expect(err).To(Not(HaveOccurred()))
```

Comment:
clean up: You can remove this block to delete FalconAdmission CR, this is already done in the `AfterEach` block.